src/components/theme-switcher/tm-switch.css

```
@@ -0,0 +1,50 @@
+oui-tm-switch {
+    position: absolute;
+}
+
+.oui-tm-switch__Light {
```

Namespace all your selectors and use lowercase:

```suggestion
oui-tm-switch .oui-tm-switch__light {
```